src/site/includes/header.html

```diff
@@ -3,80 +3,91 @@
 <!--[if IE 7 ]> <html lang="en" class="ie7"> <![endif]-->
 <!--[if IE 8 ]> <html lang="en" class="ie8"> <![endif]-->
 <!--[if IE 9 ]> <html lang="en" class="ie9"> <![endif]-->
-<!--[if (gt IE 9)|!(IE)]><!--> <html lang="en"> <!--<![endif]-->
+<!--[if (gt IE 9)|!(IE)]><!-->
+<html lang="en">
+<!--<![endif]-->
 
 <head>
-<meta charset="utf-8">
-<meta http-equiv="X-UA-Compatible" content="IE=edge" />
-<meta name="HandheldFriendly" content="True">
-<meta name="MobileOptimized" content="320">
-<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="HandheldFriendly" content="True">
+  <meta name="MobileOptimized" content="320">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-{% if drupalTags %}
+  {% if drupalTags %}
   {% include "src/site/includes/metatags.drupal.liquid" %}
-{% else %}
+  {% else %}
   {% include "src/site/includes/metatags.liquid" %}
-{% endif %}
+  {% endif %}
 
-<script src="/js/record-event.js"></script>
-<script src="/js/settings.js"></script>
+  <script src="/js/record-event.js"></script>
+  <script src="/js/settings.js"></script>
 
-{% include 'src/site/includes/google-analytics' %}
+  {% include 'src/site/includes/google-analytics' %}
 
-<!-- Icons -->
-<link href="/img/design/icons/apple-touch-icon.png" rel="apple-touch-icon-precomposed" />
-<link href="/img/design/icons/apple-touch-icon-72x72.png" rel="apple-touch-icon-precomposed" sizes="72x72" />
-<link href="/img/design/icons/apple-touch-icon-114x114.png" rel="apple-touch-icon-precomposed" sizes="114x114" />
-<link href="/img/design/icons/apple-touch-icon-152x152.png" rel="apple-touch-icon-precomposed" sizes="144x144" />
-<link rel="shortcut icon" href="/img/design/icons/favicon.ico" />
+  <!-- Icons -->
+  <link href="/img/design/icons/apple-touch-icon.png" rel="apple-touch-icon-precomposed" />
+  <link href="/img/design/icons/apple-touch-icon-72x72.png" rel="apple-touch-icon-precomposed" sizes="72x72" />
+  <link href="/img/design/icons/apple-touch-icon-114x114.png" rel="apple-touch-icon-precomposed" sizes="114x114" />
+  <link href="/img/design/icons/apple-touch-icon-152x152.png" rel="apple-touch-icon-precomposed" sizes="144x144" />
+  <link rel="shortcut icon" href="/img/design/icons/favicon.ico" />
 
-<meta name="msapplication-TileImage" content="/img/design/icons/apple-touch-icon-114x114">
-<meta name="msapplication-TileColor" content="#144073">
+  <meta name="msapplication-TileImage" content="/img/design/icons/apple-touch-icon-114x114">
+  <meta name="msapplication-TileColor" content="#144073">
 
-<!-- CSS -->
-<link rel="stylesheet" data-entry-name="style.css">
-<link rel="stylesheet" data-entry-name="{{ entryname | default: 'static-pages' }}.css">
+  <!-- Preload main fonts -->
```

Comment:
maybe mention something about why we're doing this

Reply:
Done in the initial description.